Makefile

```diff
@@ -0,0 +1,26 @@
+.DEFAULT_GOAL := help
```

Comment:
Good idea creating a Makefile. Does Shopware do this as well?

1. Please add an entry in the `README.md`, which steps need to be done, in order to lint the JS files.
2. Add another entry to the `README.md`, which shows how you can configure the IDE to show these linting errors.
2. How can we define our own linting rules, ontop of Shopwares rules, if we run Shopwares ESLint?